### ISSUE TITLE
challenge: Session Establishment Test for Host Application Profile

### ISSUE THREAD

**Issue by dvsmd** [eclipse-sparkplug/sparkplug.listings#15]
Sparkplug Specification 3.0.0 and TCK 3.0.0

Product being tested: MQTT Client (GE Digital) for the Host Application Profile
https://www.ge.com/digital/documentation/mqtt/version2023/index.html

The test expects that the host application subscribe to SpBv1.0/# namespace topic after connection establishment. However, the specification does not seem to mandate the subscription to the SpBv1.0/# and is ambiguous.

Expectation: Remove the need to subscribe to SpBv1.0/# namespace from the test and check for specific subset of topics as required.

Excerpt from the specification which does not mandate the subscription to SpBv1.0/# is below.

[tck-id-host-topic-phid-birth-sub-required] The Sparkplug Host Application MUST subscribe
to its own spBv1.0/STATE/sparkplug_host_id and the appropriate spBv1.0 topic(s) immediately
after successfully connecting to the MQTT Server.
◦ An appropriate spBv1.0 topic could simply be spBv1.0/. However, it may also make sense for a
Host Application to subscribe only to a specific Sparkplug Group. For example subscribing to
spBv1.0/Group1/ is also valid. A Host Application could even issue a subscription to subscribe to
only a single Sparkplug Edge Node using this: spBv1.0/Group1/+/EdgeNode1/#. A Sparkplug Host
Application could subscribe to a combination of specific Sparkplug Groups and/or Edge Nodes
as well.

Another area which seems to mandate the subscription to SpBv1.0/# is below.

With the MQTT Session established, Sparkplug Host Application MUST issue an MQTT subscription
for the defined Sparkplug Topic Namespace.
[tck-id-message-flow-phid-sparkplug-subscription] The subscription on the Sparkplug Topic
Namespace and the STATE topic MUST be done immediately after successfully establishing
the MQTT session and before publishing its own STATE message.

Test Log:
[Test Output.txt](https://github.com/eclipse-sparkplug/sparkplug.listings/files/11471360/Test.Output.txt)

**Comment by wes-johnson** [comment]
I agree this is a problem. I've just created this to get it fixed: https://github.com/eclipse-sparkplug/sparkplug/issues/460
